<<<<<<< HEAD
2016/05/20 09:17:17 [error] [exception.ParseError] ParseError: syntax error, unexpected 'public' (T_PUBLIC), expecting '{' in C:\xampp\htdocs\siadm\protected\models\PemasukkanBos.php:19
Stack trace:
#0 [internal function]: YiiBase::autoload('PemasukkanBos')
#1 C:\xampp\htdocs\yii\db\ar\CActiveRecord.php(395): spl_autoload_call('PemasukkanBos')
#2 C:\xampp\htdocs\yii\web\CActiveDataProvider.php(165): CActiveRecord::model('PemasukkanBos')
#3 C:\xampp\htdocs\yii\web\CActiveDataProvider.php(87): CActiveDataProvider->getModel('PemasukkanBos')
#4 C:\xampp\htdocs\siadm\protected\controllers\PemasukkanBosController.php(127): CActiveDataProvider->__construct('PemasukkanBos')
#5 C:\xampp\htdocs\yii\web\actions\CInlineAction.php(49): PemasukkanBosController->actionIndex()
#6 C:\xampp\htdocs\yii\web\CController.php(308): CInlineAction->runWithParams(Array)
#7 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#8 C:\xampp\htdocs\yii\web\filters\CFilter.php(40): CFilterChain->run()
#9 C:\xampp\htdocs\yii\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\yii\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#11 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#12 C:\xampp\htdocs\yii\web\CController.php(291): CFilterChain->run()
#13 C:\xampp\htdocs\yii\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#14 C:\xampp\htdocs\yii\web\CWebApplication.php(282): CController->run('')
#15 C:\xampp\htdocs\yii\web\CWebApplication.php(141): CWebApplication->runController('PemasukkanBos')
#16 C:\xampp\htdocs\yii\base\CApplication.php(185): CWebApplication->processRequest()
#17 C:\xampp\htdocs\siadm\index.php(33): CApplication->run()
#18 {main}
REQUEST_URI=/siadm/index.php/PemasukkanBos
HTTP_REFERER=http://localhost/siadm/index.php/Donatur
---
2016/05/20 09:17:19 [error] [exception.ParseError] ParseError: syntax error, unexpected 'public' (T_PUBLIC), expecting '{' in C:\xampp\htdocs\siadm\protected\models\PemasukkanBos.php:19
Stack trace:
#0 [internal function]: YiiBase::autoload('PemasukkanBos')
#1 C:\xampp\htdocs\yii\db\ar\CActiveRecord.php(395): spl_autoload_call('PemasukkanBos')
#2 C:\xampp\htdocs\yii\web\CActiveDataProvider.php(165): CActiveRecord::model('PemasukkanBos')
#3 C:\xampp\htdocs\yii\web\CActiveDataProvider.php(87): CActiveDataProvider->getModel('PemasukkanBos')
#4 C:\xampp\htdocs\siadm\protected\controllers\PemasukkanBosController.php(127): CActiveDataProvider->__construct('PemasukkanBos')
#5 C:\xampp\htdocs\yii\web\actions\CInlineAction.php(49): PemasukkanBosController->actionIndex()
#6 C:\xampp\htdocs\yii\web\CController.php(308): CInlineAction->runWithParams(Array)
#7 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#8 C:\xampp\htdocs\yii\web\filters\CFilter.php(40): CFilterChain->run()
#9 C:\xampp\htdocs\yii\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\yii\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#11 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#12 C:\xampp\htdocs\yii\web\CController.php(291): CFilterChain->run()
#13 C:\xampp\htdocs\yii\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#14 C:\xampp\htdocs\yii\web\CWebApplication.php(282): CController->run('')
#15 C:\xampp\htdocs\yii\web\CWebApplication.php(141): CWebApplication->runController('PemasukkanBos')
#16 C:\xampp\htdocs\yii\base\CApplication.php(185): CWebApplication->processRequest()
#17 C:\xampp\htdocs\siadm\index.php(33): CApplication->run()
#18 {main}
REQUEST_URI=/siadm/index.php/PemasukkanBos
HTTP_REFERER=http://localhost/siadm/index.php/Donatur
---
2016/05/20 09:17:25 [error] [exception.ParseError] ParseError: syntax error, unexpected 'public' (T_PUBLIC), expecting '{' in C:\xampp\htdocs\siadm\protected\models\PemasukkanBos.php:19
Stack trace:
#0 [internal function]: YiiBase::autoload('PemasukkanBos')
#1 C:\xampp\htdocs\yii\db\ar\CActiveRecord.php(395): spl_autoload_call('PemasukkanBos')
#2 C:\xampp\htdocs\yii\web\CActiveDataProvider.php(165): CActiveRecord::model('PemasukkanBos')
#3 C:\xampp\htdocs\yii\web\CActiveDataProvider.php(87): CActiveDataProvider->getModel('PemasukkanBos')
#4 C:\xampp\htdocs\siadm\protected\controllers\PemasukkanBosController.php(127): CActiveDataProvider->__construct('PemasukkanBos')
#5 C:\xampp\htdocs\yii\web\actions\CInlineAction.php(49): PemasukkanBosController->actionIndex()
#6 C:\xampp\htdocs\yii\web\CController.php(308): CInlineAction->runWithParams(Array)
#7 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#8 C:\xampp\htdocs\yii\web\filters\CFilter.php(40): CFilterChain->run()
#9 C:\xampp\htdocs\yii\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\yii\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#11 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#12 C:\xampp\htdocs\yii\web\CController.php(291): CFilterChain->run()
#13 C:\xampp\htdocs\yii\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#14 C:\xampp\htdocs\yii\web\CWebApplication.php(282): CController->run('')
#15 C:\xampp\htdocs\yii\web\CWebApplication.php(141): CWebApplication->runController('PemasukkanBos')
#16 C:\xampp\htdocs\yii\base\CApplication.php(185): CWebApplication->processRequest()
#17 C:\xampp\htdocs\siadm\index.php(33): CApplication->run()
#18 {main}
REQUEST_URI=/siadm/index.php/PemasukkanBos
HTTP_REFERER=http://localhost/siadm/index.php/Donatur
---
2016/05/20 09:18:37 [error] [exception.CHttpException.404] CHttpException: Unable to resolve the request "UnduhTransaksiPengeluaran". in C:\xampp\htdocs\yii\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii\web\CWebApplication.php(141): CWebApplication->runController('UnduhTransaksiP...')
#1 C:\xampp\htdocs\yii\base\CApplication.php(185): CWebApplication->processRequest()
#2 C:\xampp\htdocs\siadm\index.php(33): CApplication->run()
#3 {main}
REQUEST_URI=/siadm/index.php/UnduhTransaksiPengeluaran
HTTP_REFERER=http://localhost/siadm/index.php/transaksiPengeluaran
---
2016/05/20 09:19:33 [error] [exception.ParseError] ParseError: syntax error, unexpected 'public' (T_PUBLIC), expecting '{' in C:\xampp\htdocs\siadm\protected\models\LaporanPemasukkan.php:18
Stack trace:
#0 [internal function]: YiiBase::autoload('LaporanPemasukk...')
#1 C:\xampp\htdocs\siadm\protected\controllers\UnduhController.php(92): spl_autoload_call('LaporanPemasukk...')
#2 [internal function]: UnduhController->actionExcelPemasukkan('2016-05-01', '2016-05-20')
#3 C:\xampp\htdocs\yii\web\actions\CAction.php(109): ReflectionMethod->invokeArgs(Object(UnduhController), Array)
#4 C:\xampp\htdocs\yii\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(UnduhController), Object(ReflectionMethod), Array)
#5 C:\xampp\htdocs\yii\web\CController.php(308): CInlineAction->runWithParams(Array)
#6 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#7 C:\xampp\htdocs\yii\web\filters\CFilter.php(40): CFilterChain->run()
#8 C:\xampp\htdocs\yii\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#9 C:\xampp\htdocs\yii\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#10 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#11 C:\xampp\htdocs\yii\web\CController.php(291): CFilterChain->run()
#12 C:\xampp\htdocs\yii\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#13 C:\xampp\htdocs\yii\web\CWebApplication.php(282): CController->run('excelPemasukkan')
#14 C:\xampp\htdocs\yii\web\CWebApplication.php(141): CWebApplication->runController('unduh/excelPema...')
#15 C:\xampp\htdocs\yii\base\CApplication.php(185): CWebApplication->processRequest()
#16 C:\xampp\htdocs\siadm\index.php(33): CApplication->run()
#17 {main}
REQUEST_URI=/siadm/index.php/unduh/excelPemasukkan?awal=2016-05-01&akhir=2016-05-20
HTTP_REFERER=http://localhost/siadm/index.php/Unduh/UnduhTransaksiPengeluaran
---
2016/05/20 09:22:08 [error] [exception.CException] CException: Property "TransaksiPengeluaran.ID" is not defined. in C:\xampp\htdocs\yii\base\CComponent.php:130
Stack trace:
#0 C:\xampp\htdocs\yii\db\ar\CActiveRecord.php(145): CComponent->__get('ID')
#1 C:\xampp\htdocs\siadm\protected\models\Base.php(29): CActiveRecord->__get('ID')
#2 C:\xampp\htdocs\yii\db\ar\CActiveRecord.php(1100): Base->afterSave()
#3 C:\xampp\htdocs\yii\db\ar\CActiveRecord.php(809): CActiveRecord->insert(NULL)
#4 C:\xampp\htdocs\siadm\protected\controllers\TransaksiPengeluaranController.php(75): CActiveRecord->save()
#5 C:\xampp\htdocs\yii\web\actions\CInlineAction.php(49): TransaksiPengeluaranController->actionCreate()
#6 C:\xampp\htdocs\yii\web\CController.php(308): CInlineAction->runWithParams(Array)
#7 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#8 C:\xampp\htdocs\yii\web\filters\CFilter.php(40): CFilterChain->run()
#9 C:\xampp\htdocs\yii\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\yii\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#11 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#12 C:\xampp\htdocs\yii\web\CController.php(291): CFilterChain->run()
#13 C:\xampp\htdocs\yii\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#14 C:\xampp\htdocs\yii\web\CWebApplication.php(282): CController->run('create')
#15 C:\xampp\htdocs\yii\web\CWebApplication.php(141): CWebApplication->runController('transaksiPengel...')
#16 C:\xampp\htdocs\yii\base\CApplication.php(185): CWebApplication->processRequest()
#17 C:\xampp\htdocs\siadm\index.php(33): CApplication->run()
#18 {main}
REQUEST_URI=/siadm/index.php/transaksiPengeluaran/create
HTTP_REFERER=http://localhost/siadm/index.php/transaksiPengeluaran/create
---
2016/05/20 09:23:14 [error] [exception.CException] CException: Property "TransaksiPengeluaran.id" is not defined. in C:\xampp\htdocs\yii\base\CComponent.php:130
Stack trace:
#0 C:\xampp\htdocs\yii\db\ar\CActiveRecord.php(145): CComponent->__get('id')
#1 C:\xampp\htdocs\siadm\protected\models\Base.php(29): CActiveRecord->__get('id')
#2 C:\xampp\htdocs\yii\db\ar\CActiveRecord.php(1100): Base->afterSave()
#3 C:\xampp\htdocs\yii\db\ar\CActiveRecord.php(809): CActiveRecord->insert(NULL)
#4 C:\xampp\htdocs\siadm\protected\controllers\TransaksiPengeluaranController.php(75): CActiveRecord->save()
#5 C:\xampp\htdocs\yii\web\actions\CInlineAction.php(49): TransaksiPengeluaranController->actionCreate()
#6 C:\xampp\htdocs\yii\web\CController.php(308): CInlineAction->runWithParams(Array)
#7 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#8 C:\xampp\htdocs\yii\web\filters\CFilter.php(40): CFilterChain->run()
#9 C:\xampp\htdocs\yii\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\yii\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#11 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#12 C:\xampp\htdocs\yii\web\CController.php(291): CFilterChain->run()
#13 C:\xampp\htdocs\yii\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#14 C:\xampp\htdocs\yii\web\CWebApplication.php(282): CController->run('create')
#15 C:\xampp\htdocs\yii\web\CWebApplication.php(141): CWebApplication->runController('transaksiPengel...')
#16 C:\xampp\htdocs\yii\base\CApplication.php(185): CWebApplication->processRequest()
#17 C:\xampp\htdocs\siadm\index.php(33): CApplication->run()
#18 {main}
REQUEST_URI=/siadm/index.php/transaksiPengeluaran/create
HTTP_REFERER=http://localhost/siadm/index.php/transaksiPengeluaran/create
---
2016/05/20 09:23:17 [error] [exception.CException] CException: Property "TransaksiPengeluaran.id" is not defined. in C:\xampp\htdocs\yii\base\CComponent.php:130
Stack trace:
#0 C:\xampp\htdocs\yii\db\ar\CActiveRecord.php(145): CComponent->__get('id')
#1 C:\xampp\htdocs\siadm\protected\models\Base.php(29): CActiveRecord->__get('id')
#2 C:\xampp\htdocs\yii\db\ar\CActiveRecord.php(1100): Base->afterSave()
#3 C:\xampp\htdocs\yii\db\ar\CActiveRecord.php(809): CActiveRecord->insert(NULL)
#4 C:\xampp\htdocs\siadm\protected\controllers\TransaksiPengeluaranController.php(75): CActiveRecord->save()
#5 C:\xampp\htdocs\yii\web\actions\CInlineAction.php(49): TransaksiPengeluaranController->actionCreate()
#6 C:\xampp\htdocs\yii\web\CController.php(308): CInlineAction->runWithParams(Array)
#7 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#8 C:\xampp\htdocs\yii\web\filters\CFilter.php(40): CFilterChain->run()
#9 C:\xampp\htdocs\yii\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\yii\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#11 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#12 C:\xampp\htdocs\yii\web\CController.php(291): CFilterChain->run()
#13 C:\xampp\htdocs\yii\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#14 C:\xampp\htdocs\yii\web\CWebApplication.php(282): CController->run('create')
#15 C:\xampp\htdocs\yii\web\CWebApplication.php(141): CWebApplication->runController('transaksiPengel...')
#16 C:\xampp\htdocs\yii\base\CApplication.php(185): CWebApplication->processRequest()
#17 C:\xampp\htdocs\siadm\index.php(33): CApplication->run()
#18 {main}
REQUEST_URI=/siadm/index.php/transaksiPengeluaran/create
HTTP_REFERER=http://localhost/siadm/index.php/transaksiPengeluaran/create
---
2016/05/20 09:24:14 [error] [exception.CException] CException: Property "TransaksiPengeluaran.id" is not defined. in C:\xampp\htdocs\yii\base\CComponent.php:130
Stack trace:
#0 C:\xampp\htdocs\yii\db\ar\CActiveRecord.php(145): CComponent->__get('id')
#1 C:\xampp\htdocs\siadm\protected\models\Base.php(29): CActiveRecord->__get('id')
#2 C:\xampp\htdocs\yii\db\ar\CActiveRecord.php(1100): Base->afterSave()
#3 C:\xampp\htdocs\yii\db\ar\CActiveRecord.php(809): CActiveRecord->insert(NULL)
#4 C:\xampp\htdocs\siadm\protected\controllers\TransaksiPengeluaranController.php(75): CActiveRecord->save()
#5 C:\xampp\htdocs\yii\web\actions\CInlineAction.php(49): TransaksiPengeluaranController->actionCreate()
#6 C:\xampp\htdocs\yii\web\CController.php(308): CInlineAction->runWithParams(Array)
#7 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#8 C:\xampp\htdocs\yii\web\filters\CFilter.php(40): CFilterChain->run()
#9 C:\xampp\htdocs\yii\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\yii\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#11 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#12 C:\xampp\htdocs\yii\web\CController.php(291): CFilterChain->run()
#13 C:\xampp\htdocs\yii\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#14 C:\xampp\htdocs\yii\web\CWebApplication.php(282): CController->run('create')
#15 C:\xampp\htdocs\yii\web\CWebApplication.php(141): CWebApplication->runController('transaksiPengel...')
#16 C:\xampp\htdocs\yii\base\CApplication.php(185): CWebApplication->processRequest()
#17 C:\xampp\htdocs\siadm\index.php(33): CApplication->run()
#18 {main}
REQUEST_URI=/siadm/index.php/transaksiPengeluaran/create
HTTP_REFERER=http://localhost/siadm/index.php/transaksiPengeluaran/create
---
2016/05/20 09:25:28 [error] [exception.CException] CException: Property "TransaksiPengeluaran.id" is not defined. in C:\xampp\htdocs\yii\base\CComponent.php:130
Stack trace:
#0 C:\xampp\htdocs\yii\db\ar\CActiveRecord.php(145): CComponent->__get('id')
#1 C:\xampp\htdocs\siadm\protected\models\Base.php(29): CActiveRecord->__get('id')
#2 C:\xampp\htdocs\yii\db\ar\CActiveRecord.php(1100): Base->afterSave()
#3 C:\xampp\htdocs\yii\db\ar\CActiveRecord.php(809): CActiveRecord->insert(NULL)
#4 C:\xampp\htdocs\siadm\protected\controllers\TransaksiPengeluaranController.php(75): CActiveRecord->save()
#5 C:\xampp\htdocs\yii\web\actions\CInlineAction.php(49): TransaksiPengeluaranController->actionCreate()
#6 C:\xampp\htdocs\yii\web\CController.php(308): CInlineAction->runWithParams(Array)
#7 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#8 C:\xampp\htdocs\yii\web\filters\CFilter.php(40): CFilterChain->run()
#9 C:\xampp\htdocs\yii\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\yii\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#11 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#12 C:\xampp\htdocs\yii\web\CController.php(291): CFilterChain->run()
#13 C:\xampp\htdocs\yii\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#14 C:\xampp\htdocs\yii\web\CWebApplication.php(282): CController->run('create')
#15 C:\xampp\htdocs\yii\web\CWebApplication.php(141): CWebApplication->runController('transaksiPengel...')
#16 C:\xampp\htdocs\yii\base\CApplication.php(185): CWebApplication->processRequest()
#17 C:\xampp\htdocs\siadm\index.php(33): CApplication->run()
#18 {main}
REQUEST_URI=/siadm/index.php/transaksiPengeluaran/create
HTTP_REFERER=http://localhost/siadm/index.php/transaksiPengeluaran/create
---
2016/05/20 09:28:14 [error] [exception.CException] CException: Class name "Log" does not match class file "C:\xampp\htdocs\siadm\protected\models\Log.php". in C:\xampp\htdocs\yii\YiiBase.php:423
Stack trace:
#0 [internal function]: YiiBase::autoload('Log')
#1 C:\xampp\htdocs\siadm\protected\models\Base.php(14): spl_autoload_call('Log')
#2 C:\xampp\htdocs\siadm\protected\models\Base.php(29): Base->log('Create Transaks...')
#3 C:\xampp\htdocs\yii\db\ar\CActiveRecord.php(1100): Base->afterSave()
#4 C:\xampp\htdocs\yii\db\ar\CActiveRecord.php(809): CActiveRecord->insert(NULL)
#5 C:\xampp\htdocs\siadm\protected\controllers\TransaksiPengeluaranController.php(75): CActiveRecord->save()
#6 C:\xampp\htdocs\yii\web\actions\CInlineAction.php(49): TransaksiPengeluaranController->actionCreate()
#7 C:\xampp\htdocs\yii\web\CController.php(308): CInlineAction->runWithParams(Array)
#8 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#9 C:\xampp\htdocs\yii\web\filters\CFilter.php(40): CFilterChain->run()
#10 C:\xampp\htdocs\yii\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#11 C:\xampp\htdocs\yii\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#12 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#13 C:\xampp\htdocs\yii\web\CController.php(291): CFilterChain->run()
#14 C:\xampp\htdocs\yii\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#15 C:\xampp\htdocs\yii\web\CWebApplication.php(282): CController->run('create')
#16 C:\xampp\htdocs\yii\web\CWebApplication.php(141): CWebApplication->runController('transaksiPengel...')
#17 C:\xampp\htdocs\yii\base\CApplication.php(185): CWebApplication->processRequest()
#18 C:\xampp\htdocs\siadm\index.php(33): CApplication->run()
#19 {main}
REQUEST_URI=/siadm/index.php/transaksiPengeluaran/create
HTTP_REFERER=http://localhost/siadm/index.php/transaksiPengeluaran/create
---
2016/05/20 09:28:59 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'siadmdb.log' doesn't exist. The SQL statement executed was: SHOW FULL COLUMNS FROM `log`.
in C:\xampp\htdocs\siadm\protected\models\Base.php (14)
in C:\xampp\htdocs\siadm\protected\models\Base.php (29)
in C:\xampp\htdocs\siadm\protected\controllers\TransaksiPengeluaranController.php (75)
2016/05/20 09:28:59 [error] [exception.CDbException] CDbException: The table "log" for active record class "Log" cannot be found in the database. in C:\xampp\htdocs\yii\db\ar\CActiveRecord.php:2387
Stack trace:
#0 C:\xampp\htdocs\yii\db\ar\CActiveRecord.php(411): CActiveRecordMetaData->__construct(Object(Log))
#1 C:\xampp\htdocs\yii\db\ar\CActiveRecord.php(79): CActiveRecord->getMetaData()
#2 C:\xampp\htdocs\siadm\protected\models\Base.php(14): CActiveRecord->__construct()
#3 C:\xampp\htdocs\siadm\protected\models\Base.php(29): Base->log('Create Transaks...')
#4 C:\xampp\htdocs\yii\db\ar\CActiveRecord.php(1100): Base->afterSave()
#5 C:\xampp\htdocs\yii\db\ar\CActiveRecord.php(809): CActiveRecord->insert(NULL)
#6 C:\xampp\htdocs\siadm\protected\controllers\TransaksiPengeluaranController.php(75): CActiveRecord->save()
#7 C:\xampp\htdocs\yii\web\actions\CInlineAction.php(49): TransaksiPengeluaranController->actionCreate()
#8 C:\xampp\htdocs\yii\web\CController.php(308): CInlineAction->runWithParams(Array)
#9 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#10 C:\xampp\htdocs\yii\web\filters\CFilter.php(40): CFilterChain->run()
#11 C:\xampp\htdocs\yii\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#12 C:\xampp\htdocs\yii\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#13 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#14 C:\xampp\htdocs\yii\web\CController.php(291): CFilterChain->run()
#15 C:\xampp\htdocs\yii\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#16 C:\xampp\htdocs\yii\web\CWebApplication.php(282): CController->run('create')
#17 C:\xampp\htdocs\yii\web\CWebApplication.php(141): CWebApplication->runController('transaksiPengel...')
#18 C:\xampp\htdocs\yii\base\CApplication.php(185): CWebApplication->processRequest()
#19 C:\xampp\htdocs\siadm\index.php(33): CApplication->run()
#20 {main}
REQUEST_URI=/siadm/index.php/transaksiPengeluaran/create
HTTP_REFERER=http://localhost/siadm/index.php/transaksiPengeluaran/create
---
2016/05/20 05:30:35 [error] [exception.CException] exception 'CException' with message 'Alias "chartjs.components.ChartJs" is invalid. Make sure it points to an existing PHP file and the file is readable.' in C:\xampp\htdocs\yii\YiiBase.php:323
Stack trace:
#0 C:\xampp\htdocs\yii\YiiBase.php(196): YiiBase::import('chartjs.compone...', true)
#1 C:\xampp\htdocs\yii\base\CModule.php(393): YiiBase::createComponent(Array)
#2 C:\xampp\htdocs\yii\base\CModule.php(530): CModule->getComponent('chartjs')
#3 C:\xampp\htdocs\yii\base\CApplication.php(168): CModule->preloadComponents()
#4 C:\xampp\htdocs\yii\YiiBase.php(125): CApplication->__construct('C:\\xampp\\htdocs...')
#5 C:\xampp\htdocs\yii\YiiBase.php(98): YiiBase::createApplication('CWebApplication', 'C:\\xampp\\htdocs...')
#6 C:\xampp\htdocs\siadm\index.php(17): YiiBase::createWebApplication('C:\\xampp\\htdocs...')
#7 {main}
REQUEST_URI=/siadm/index.php
---
2016/05/20 05:59:58 [error] [exception.CException] exception 'CException' with message 'Alias "chartjs.components.ChartJs" is invalid. Make sure it points to an existing PHP file and the file is readable.' in C:\xampp\htdocs\yii\YiiBase.php:323
Stack trace:
#0 C:\xampp\htdocs\yii\YiiBase.php(196): YiiBase::import('chartjs.compone...', true)
#1 C:\xampp\htdocs\yii\base\CModule.php(393): YiiBase::createComponent(Array)
#2 C:\xampp\htdocs\yii\base\CModule.php(530): CModule->getComponent('chartjs')
#3 C:\xampp\htdocs\yii\base\CApplication.php(168): CModule->preloadComponents()
#4 C:\xampp\htdocs\yii\YiiBase.php(125): CApplication->__construct('C:\\xampp\\htdocs...')
#5 C:\xampp\htdocs\yii\YiiBase.php(98): YiiBase::createApplication('CWebApplication', 'C:\\xampp\\htdocs...')
#6 C:\xampp\htdocs\siadm\index.php(17): YiiBase::createWebApplication('C:\\xampp\\htdocs...')
#7 {main}
REQUEST_URI=/siadm/index.php
---
2016/05/20 06:03:34 [error] [exception.CException] exception 'CException' with message 'Class name "Log" does not match class file "C:\xampp\htdocs\siadm\protected\models\Log.php".' in C:\xampp\htdocs\yii\YiiBase.php:423
Stack trace:
#0 [internal function]: YiiBase::autoload('Log')
#1 C:\xampp\htdocs\siadm\protected\models\Base.php(14): spl_autoload_call('Log')
#2 C:\xampp\htdocs\siadm\protected\models\Base.php(29): Base->log('Create User dat...')
#3 C:\xampp\htdocs\yii\db\ar\CActiveRecord.php(1100): Base->afterSave()
#4 C:\xampp\htdocs\yii\db\ar\CActiveRecord.php(809): CActiveRecord->insert(NULL)
#5 C:\xampp\htdocs\siadm\protected\controllers\UserController.php(141): CActiveRecord->save()
#6 C:\xampp\htdocs\yii\web\actions\CInlineAction.php(49): UserController->actionIndex()
#7 C:\xampp\htdocs\yii\web\CController.php(308): CInlineAction->runWithParams(Array)
#8 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#9 C:\xampp\htdocs\yii\web\filters\CFilter.php(40): CFilterChain->run()
#10 C:\xampp\htdocs\yii\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#11 C:\xampp\htdocs\yii\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#12 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#13 C:\xampp\htdocs\yii\web\CController.php(291): CFilterChain->run()
#14 C:\xampp\htdocs\yii\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#15 C:\xampp\htdocs\yii\web\CWebApplication.php(282): CController->run('')
#16 C:\xampp\htdocs\yii\web\CWebApplication.php(141): CWebApplication->runController('User')
#17 C:\xampp\htdocs\yii\base\CApplication.php(185): CWebApplication->processRequest()
#18 C:\xampp\htdocs\siadm\index.php(33): CApplication->run()
#19 {main}
REQUEST_URI=/siadm/index.php/User
HTTP_REFERER=http://localhost/siadm/index.php/User
---
2016/05/20 06:04:38 [error] [exception.CException] exception 'CException' with message 'Class name "Log" does not match class file "C:\xampp\htdocs\siadm\protected\models\Log.php".' in C:\xampp\htdocs\yii\YiiBase.php:423
Stack trace:
#0 [internal function]: YiiBase::autoload('Log')
#1 C:\xampp\htdocs\siadm\protected\models\Base.php(14): spl_autoload_call('Log')
#2 C:\xampp\htdocs\siadm\protected\models\Base.php(29): Base->log('Create User dat...')
#3 C:\xampp\htdocs\yii\db\ar\CActiveRecord.php(1100): Base->afterSave()
#4 C:\xampp\htdocs\yii\db\ar\CActiveRecord.php(809): CActiveRecord->insert(NULL)
#5 C:\xampp\htdocs\siadm\protected\controllers\UserController.php(141): CActiveRecord->save()
#6 C:\xampp\htdocs\yii\web\actions\CInlineAction.php(49): UserController->actionIndex()
#7 C:\xampp\htdocs\yii\web\CController.php(308): CInlineAction->runWithParams(Array)
#8 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#9 C:\xampp\htdocs\yii\web\filters\CFilter.php(40): CFilterChain->run()
#10 C:\xampp\htdocs\yii\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#11 C:\xampp\htdocs\yii\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#12 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#13 C:\xampp\htdocs\yii\web\CController.php(291): CFilterChain->run()
#14 C:\xampp\htdocs\yii\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#15 C:\xampp\htdocs\yii\web\CWebApplication.php(282): CController->run('')
#16 C:\xampp\htdocs\yii\web\CWebApplication.php(141): CWebApplication->runController('User')
#17 C:\xampp\htdocs\yii\base\CApplication.php(185): CWebApplication->processRequest()
#18 C:\xampp\htdocs\siadm\index.php(33): CApplication->run()
#19 {main}
REQUEST_URI=/siadm/index.php/User
HTTP_REFERER=http://localhost/siadm/index.php/User
---
2016/05/20 06:33:55 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "unduhNilai".' in C:\xampp\htdocs\yii\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii\web\CWebApplication.php(141): CWebApplication->runController('unduhNilai')
#1 C:\xampp\htdocs\yii\base\CApplication.php(185): CWebApplication->processRequest()
#2 C:\xampp\htdocs\siadm\index.php(33): CApplication->run()
#3 {main}
REQUEST_URI=/siadm/index.php/unduhNilai
HTTP_REFERER=http://localhost/siadm/index.php/MataPelajaran
---
2016/05/20 06:34:52 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "unduhRapor".' in C:\xampp\htdocs\yii\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii\web\CWebApplication.php(141): CWebApplication->runController('unduhRapor')
#1 C:\xampp\htdocs\yii\base\CApplication.php(185): CWebApplication->processRequest()
#2 C:\xampp\htdocs\siadm\index.php(33): CApplication->run()
#3 {main}
REQUEST_URI=/siadm/index.php/unduhRapor
HTTP_REFERER=http://localhost/siadm/index.php/MataPelajaran
---
2016/05/20 06:46:58 [error] [exception.CException] exception 'CException' with message 'Property "RiwayatPenyakit.id" is not defined.' in C:\xampp\htdocs\yii\base\CComponent.php:130
Stack trace:
#0 C:\xampp\htdocs\yii\db\ar\CActiveRecord.php(145): CComponent->__get('id')
#1 C:\xampp\htdocs\siadm\protected\models\Base.php(29): CActiveRecord->__get('id')
#2 C:\xampp\htdocs\yii\db\ar\CActiveRecord.php(1100): Base->afterSave()
#3 C:\xampp\htdocs\yii\db\ar\CActiveRecord.php(809): CActiveRecord->insert(NULL)
#4 C:\xampp\htdocs\siadm\protected\controllers\RiwayatPenyakitController.php(82): CActiveRecord->save()
#5 [internal function]: RiwayatPenyakitController->actionCreate(NULL)
#6 C:\xampp\htdocs\yii\web\actions\CAction.php(109): ReflectionMethod->invokeArgs(Object(RiwayatPenyakitController), Array)
#7 C:\xampp\htdocs\yii\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(RiwayatPenyakitController), Object(ReflectionMethod), Array)
#8 C:\xampp\htdocs\yii\web\CController.php(308): CInlineAction->runWithParams(Array)
#9 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#10 C:\xampp\htdocs\yii\web\filters\CFilter.php(40): CFilterChain->run()
#11 C:\xampp\htdocs\yii\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#12 C:\xampp\htdocs\yii\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#13 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#14 C:\xampp\htdocs\yii\web\CController.php(291): CFilterChain->run()
#15 C:\xampp\htdocs\yii\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#16 C:\xampp\htdocs\yii\web\CWebApplication.php(282): CController->run('create')
#17 C:\xampp\htdocs\yii\web\CWebApplication.php(141): CWebApplication->runController('RiwayatPenyakit...')
#18 C:\xampp\htdocs\yii\base\CApplication.php(185): CWebApplication->processRequest()
#19 C:\xampp\htdocs\siadm\index.php(33): CApplication->run()
#20 {main}
REQUEST_URI=/siadm/index.php/RiwayatPenyakit/create
HTTP_REFERER=http://localhost/siadm/index.php/RiwayatPenyakit/create
---
=======
>>>>>>> e283c1018289942129056fb71a391b5298f04942
2016/05/20 06:50:42 [error] [exception.CException] exception 'CException' with message 'Class name "Log" does not match class file "C:\xampp\htdocs\siadm\protected\models\Log.php".' in C:\xampp\htdocs\yii\YiiBase.php:423
Stack trace:
#0 [internal function]: YiiBase::autoload('Log')
#1 C:\xampp\htdocs\siadm\protected\models\Base.php(14): spl_autoload_call('Log')
#2 C:\xampp\htdocs\siadm\protected\models\Base.php(29): Base->log('Create RiwayatP...')
#3 C:\xampp\htdocs\yii\db\ar\CActiveRecord.php(1100): Base->afterSave()
#4 C:\xampp\htdocs\yii\db\ar\CActiveRecord.php(809): CActiveRecord->insert(NULL)
#5 C:\xampp\htdocs\siadm\protected\controllers\RiwayatPenyakitController.php(82): CActiveRecord->save()
#6 [internal function]: RiwayatPenyakitController->actionCreate(NULL)
#7 C:\xampp\htdocs\yii\web\actions\CAction.php(109): ReflectionMethod->invokeArgs(Object(RiwayatPenyakitController), Array)
#8 C:\xampp\htdocs\yii\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(RiwayatPenyakitController), Object(ReflectionMethod), Array)
#9 C:\xampp\htdocs\yii\web\CController.php(308): CInlineAction->runWithParams(Array)
#10 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#11 C:\xampp\htdocs\yii\web\filters\CFilter.php(40): CFilterChain->run()
#12 C:\xampp\htdocs\yii\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#13 C:\xampp\htdocs\yii\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#14 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#15 C:\xampp\htdocs\yii\web\CController.php(291): CFilterChain->run()
#16 C:\xampp\htdocs\yii\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#17 C:\xampp\htdocs\yii\web\CWebApplication.php(282): CController->run('create')
#18 C:\xampp\htdocs\yii\web\CWebApplication.php(141): CWebApplication->runController('RiwayatPenyakit...')
#19 C:\xampp\htdocs\yii\base\CApplication.php(185): CWebApplication->processRequest()
#20 C:\xampp\htdocs\siadm\index.php(33): CApplication->run()
#21 {main}
REQUEST_URI=/siadm/index.php/RiwayatPenyakit/create
HTTP_REFERER=http://localhost/siadm/index.php/RiwayatPenyakit/create
---
2016/05/20 07:04:46 [error] [exception.CException] exception 'CException' with message 'Property "PencatatanPerizinan.id" is not defined.' in C:\xampp\htdocs\yii\base\CComponent.php:130
Stack trace:
#0 C:\xampp\htdocs\yii\db\ar\CActiveRecord.php(145): CComponent->__get('id')
#1 C:\xampp\htdocs\yii\validators\CUniqueValidator.php(82): CActiveRecord->__get('id')
#2 C:\xampp\htdocs\yii\validators\CValidator.php(201): CUniqueValidator->validateAttribute(Object(PencatatanPerizinan), 'id')
#3 C:\xampp\htdocs\yii\base\CModel.php(159): CValidator->validate(Object(PencatatanPerizinan), NULL)
#4 C:\xampp\htdocs\yii\db\ar\CActiveRecord.php(808): CModel->validate(NULL)
#5 C:\xampp\htdocs\siadm\protected\controllers\PencatatanPerizinanController.php(85): CActiveRecord->save()
#6 [internal function]: PencatatanPerizinanController->actionCreate(NULL)
#7 C:\xampp\htdocs\yii\web\actions\CAction.php(109): ReflectionMethod->invokeArgs(Object(PencatatanPerizinanController), Array)
#8 C:\xampp\htdocs\yii\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(PencatatanPerizinanController), Object(ReflectionMethod), Array)
#9 C:\xampp\htdocs\yii\web\CController.php(308): CInlineAction->runWithParams(Array)
#10 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#11 C:\xampp\htdocs\yii\web\filters\CFilter.php(40): CFilterChain->run()
#12 C:\xampp\htdocs\yii\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#13 C:\xampp\htdocs\yii\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#14 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#15 C:\xampp\htdocs\yii\web\CController.php(291): CFilterChain->run()
#16 C:\xampp\htdocs\yii\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#17 C:\xampp\htdocs\yii\web\CWebApplication.php(282): CController->run('create')
#18 C:\xampp\htdocs\yii\web\CWebApplication.php(141): CWebApplication->runController('PencatatanPeriz...')
#19 C:\xampp\htdocs\yii\base\CApplication.php(185): CWebApplication->processRequest()
#20 C:\xampp\htdocs\siadm\index.php(33): CApplication->run()
#21 {main}
REQUEST_URI=/siadm/index.php/PencatatanPerizinan/create
HTTP_REFERER=http://localhost/siadm/index.php/PencatatanPerizinan/create
---
2016/05/20 07:05:17 [error] [exception.CException] exception 'CException' with message 'Property "PencatatanPerizinan.id" is not defined.' in C:\xampp\htdocs\yii\base\CComponent.php:130
Stack trace:
#0 C:\xampp\htdocs\yii\db\ar\CActiveRecord.php(145): CComponent->__get('id')
#1 C:\xampp\htdocs\yii\validators\CUniqueValidator.php(82): CActiveRecord->__get('id')
#2 C:\xampp\htdocs\yii\validators\CValidator.php(201): CUniqueValidator->validateAttribute(Object(PencatatanPerizinan), 'id')
#3 C:\xampp\htdocs\yii\base\CModel.php(159): CValidator->validate(Object(PencatatanPerizinan), NULL)
#4 C:\xampp\htdocs\yii\db\ar\CActiveRecord.php(808): CModel->validate(NULL)
#5 C:\xampp\htdocs\siadm\protected\controllers\PencatatanPerizinanController.php(85): CActiveRecord->save()
#6 [internal function]: PencatatanPerizinanController->actionCreate(NULL)
#7 C:\xampp\htdocs\yii\web\actions\CAction.php(109): ReflectionMethod->invokeArgs(Object(PencatatanPerizinanController), Array)
#8 C:\xampp\htdocs\yii\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(PencatatanPerizinanController), Object(ReflectionMethod), Array)
#9 C:\xampp\htdocs\yii\web\CController.php(308): CInlineAction->runWithParams(Array)
#10 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#11 C:\xampp\htdocs\yii\web\filters\CFilter.php(40): CFilterChain->run()
#12 C:\xampp\htdocs\yii\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#13 C:\xampp\htdocs\yii\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#14 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#15 C:\xampp\htdocs\yii\web\CController.php(291): CFilterChain->run()
#16 C:\xampp\htdocs\yii\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#17 C:\xampp\htdocs\yii\web\CWebApplication.php(282): CController->run('create')
#18 C:\xampp\htdocs\yii\web\CWebApplication.php(141): CWebApplication->runController('PencatatanPeriz...')
#19 C:\xampp\htdocs\yii\base\CApplication.php(185): CWebApplication->processRequest()
#20 C:\xampp\htdocs\siadm\index.php(33): CApplication->run()
#21 {main}
REQUEST_URI=/siadm/index.php/PencatatanPerizinan/create
HTTP_REFERER=http://localhost/siadm/index.php/PencatatanPerizinan/create
---
2016/05/20 07:08:33 [error] [exception.CException] exception 'CException' with message 'Property "PencatatanPerizinan.id" is not defined.' in C:\xampp\htdocs\yii\base\CComponent.php:130
Stack trace:
#0 C:\xampp\htdocs\yii\db\ar\CActiveRecord.php(145): CComponent->__get('id')
#1 C:\xampp\htdocs\yii\validators\CUniqueValidator.php(82): CActiveRecord->__get('id')
#2 C:\xampp\htdocs\yii\validators\CValidator.php(201): CUniqueValidator->validateAttribute(Object(PencatatanPerizinan), 'id')
#3 C:\xampp\htdocs\yii\base\CModel.php(159): CValidator->validate(Object(PencatatanPerizinan), NULL)
#4 C:\xampp\htdocs\yii\db\ar\CActiveRecord.php(808): CModel->validate(NULL)
#5 C:\xampp\htdocs\siadm\protected\controllers\PencatatanPerizinanController.php(85): CActiveRecord->save()
#6 [internal function]: PencatatanPerizinanController->actionCreate(NULL)
#7 C:\xampp\htdocs\yii\web\actions\CAction.php(109): ReflectionMethod->invokeArgs(Object(PencatatanPerizinanController), Array)
#8 C:\xampp\htdocs\yii\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(PencatatanPerizinanController), Object(ReflectionMethod), Array)
#9 C:\xampp\htdocs\yii\web\CController.php(308): CInlineAction->runWithParams(Array)
#10 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#11 C:\xampp\htdocs\yii\web\filters\CFilter.php(40): CFilterChain->run()
#12 C:\xampp\htdocs\yii\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#13 C:\xampp\htdocs\yii\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#14 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#15 C:\xampp\htdocs\yii\web\CController.php(291): CFilterChain->run()
#16 C:\xampp\htdocs\yii\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#17 C:\xampp\htdocs\yii\web\CWebApplication.php(282): CController->run('create')
#18 C:\xampp\htdocs\yii\web\CWebApplication.php(141): CWebApplication->runController('PencatatanPeriz...')
#19 C:\xampp\htdocs\yii\base\CApplication.php(185): CWebApplication->processRequest()
#20 C:\xampp\htdocs\siadm\index.php(33): CApplication->run()
#21 {main}
REQUEST_URI=/siadm/index.php/PencatatanPerizinan/create
HTTP_REFERER=http://localhost/siadm/index.php/PencatatanPerizinan/create
---
